homeassistant/components/hydrawise/sensor.py

```diff
@@ -168,29 +168,30 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up the Hydrawise sensor platform."""
-    coordinator: HydrawiseDataUpdateCoordinator = hass.data[DOMAIN][
-        config_entry.entry_id
-    ]
+    coordinators: HydrawiseUpdateCoordinators = hass.data[DOMAIN][config_entry.entry_id]
     entities: list[HydrawiseSensor] = []
-    for controller in coordinator.data.controllers.values():
+    for controller in coordinators.main.data.controllers.values():
         entities.extend(
-            HydrawiseSensor(coordinator, description, controller)
+            HydrawiseSensor(coordinators.main, description, controller)
             for description in CONTROLLER_SENSORS
```

Comment:
You will have to split `CONTROLLER_SENSORS` and `ZONE_SENSORS` now because all of the daily active watering *time* entities also rely on the water flow API call.